auto_ria/auto_ria/items.csv

```diff
@@ -0,0 +1,17 @@
+Link,Brand,Model,Name,Phone,Price,City,Mileage,Year

```

Comment:
Is this the output file, generated by your scraper? If so, is it needed under source control?

Reply:
Agree, removed .csv and .jl files